oauth2_http/java/com/google/auth/oauth2/CredentialAccessBoundary.java

```diff
@@ -120,6 +120,9 @@ private Builder() {}
      * Sets the list of {@link AccessBoundaryRule}'s.
      *
      * <p>This list must not exceed 10 rules.
+     *
+     * @param rule the rules to be set.
```

Comment:
the rules = a collection of rules (otherwise looks funny: "rule the rules")
Also, you can mention that this should not be null